packages/api/src/routes/mint.ts

```diff
@@ -86,13 +91,139 @@ const mint: FastifyPluginAsync = async (fastify, opts): Promise<void> => {
       }
 
       // Build message to sign
-      const encodedPlayerMedals = encodeMedals(player.medals)
-      const encodedRanchMedals = encodeMedals(player.medals)
-      const message = web3.utils.encodePacked(
-        request.body.address,
-        player.points,
-        encodedPlayerMedals,
-        encodedRanchMedals
+      /*
+        function mintFarmerAwards(
+            address _tokenOwner,
+            uint256 _ranchId,
+            uint256 _farmerId,
+            uint256 _farmerScore,
+            string memory _farmerName,
+            WittyBufficorns.Award[] calldata _farmerAwards,
+            bytes memory _signature
+        )
+        public
+        virtual override
+        nonReentrant
+        inStatus(WittyBufficorns.Status.Awarding)
+    {
+       */
+      // Fake values for testing
+      /*
+      const ranchId = 0
+      const farmerId = 7
+      const farmerScore = 12345
+      const farmerName = 'farmer.7'
+      const farmerAwards = [
+        {
+          category: 0,
+          ranking: 1,
+          bufficornId: 0,
+        },
+        {
+          category: 1,
+          ranking: 3,
+          bufficornId: 0,
+        },
+      ]
+     */
+      /*
+      enum Awards {
+          BestBreeder,
+          BestRanch,
+
+          BestBufficorn,
+          CoolestBufficorn,
+          FastestBufficorn,
+          MostVigorousBufficorn,
+          MostEnduringBufficorn,
+          MostAgileBufficorn,
+          WarmestBufficorn
+      }
+       */
```

Comment:
Those comments should be beautified to explain that they are the requirements from the smart contract